src/onegov/swissvotes/models/actor.py

```diff
@@ -213,9 +213,10 @@ def label(self) -> str:
             'vsa': _("actor-vsa-label"),
         }.get(self.name, self.name)
 
-    # FIXME: Use Markup
     def html(self, request: 'SwissvotesRequest') -> str:
-        return '<span title="{}">{}</span>'.format(
-            request.translate(self.label),
-            request.translate(self.abbreviation)
+        return Markup(  # noqa: MS001
+            '<span title="{}">{}</span>'.format(
+                request.translate(self.label),
+                request.translate(self.abbreviation)
+            )
```

Comment:
```suggestion
        return Markup('<span title="{}">{}</span>').format(
            request.translate(self.label),
            request.translate(self.abbreviation)
        )
```
Auch hier, MS001 ist genau da um diesen Fehler abzufangen, da so gar nichts escaped wird.

Reply:
Hoppla, hab grad nicht viel überlegt